### PR TITLE
Add guitar model and play button to welcome page

### LINKED ISSUE #16: Add a guitar model to the scene.
Need a customized 3D guitar to rotate and move in the scene.

We can use this one for example:
https://market.pmnd.rs/model/guitar

The guitar will have to match the website color templates using tools such as blender or directly from the code if we want to animate its colors.

Also, optimize the guitar using blender.

### PR DESCRIPTION
**Pull request by Hicham2012**
I added a guitar model and a play button that we will edit later on to match our design.

### CODE REVIEW

File: src/modules/Experience/components/Welcome/Welcome.tsx [hunk 1]
```diff
@@ -24,16 +28,30 @@ const Welcome = ({ position }: WelcomeProps) => {
...
         scale={active ? 1.5 : 1}
         onClick={(event) => setActive(!active)}
         onPointerOver={(event) => setHover(true)}
-        onPointerOut={(event) => setHover(false)}>
+        onPointerOut={(event) => setHover(false)}
+      >
         <boxGeometry args={[1, 1, 1]} />
         <meshStandardMaterial color={hovered ? 'hotpink' : 'orange'} />
-      </mesh>
+      </mesh> */}
```
Comment (tkalejandro) [inline src/modules/Experience/components/Welcome/Welcome.tsx]: You can delete this and everything is not been used. This was dummy

File: src/modules/Experience/components/Welcome/Welcome.tsx [hunk 2]
```diff
@@ -13,6 +13,10 @@ const Welcome = ({ position }: WelcomeProps) => {
   // Set up state for the hovered and active state
   const [hovered, setHover] = useState<boolean>(false)
   const [active, setActive] = useState<boolean>(false)
+
+  // Import the guitar
+  const guitar = useGLTF<string>('https://vazxmixjsiawhamofees.supabase.co/storage/v1/object/public/models/guitar/model.gltf')
+
```
Comment (tkalejandro) [inline src/modules/Experience/components/Welcome/Welcome.tsx:19]: FYI: This is okay for now, but I think is better in the future to have all models inside in the project and not been dependent of URLs.  I will  create the strategy to this inside the ticket with SSR, that I made.

File: src/styles/globals.css [hunk 3]
```diff
@@ -26,16 +26,21 @@ body {
...
     height: 100%;
     background: ivory;
+}
+
+.play-button>div {
+    padding: 10px;
+    border: 1px solid;
+    border-color: black;
+    cursor: pointer;
+    /* background-color: red; */
```
Comment (tkalejandro) [inline src/styles/globals.css]: This is okay for now,  once we have the UI working, we will probably handle all CSS in the TS files.  First we need to check how powerful is NextUI. 

FYI ->  We use global CSS for VERY specific things.  Having a huge CSS file is not sustainable and not flexible. We could SCSS but its a bit annoying to work with logic. Any JS CSS base solution will be always better.   In the worst case scenario, we could use CSS modules (is a NEXT js thing) better than CSS.

File: src/modules/Experience/components/Welcome/Welcome.tsx [hunk 4]
```diff
@@ -24,16 +28,30 @@ const Welcome = ({ position }: WelcomeProps) => {
...
+          wrapperClass='play-button'
+          position={[0, -0.75, 0]}
+          scale={0}
+        >
+          Play
+        </Html>
+      </Center>
       <Text
         position={[0, -1, 0]}
         scale={0.2}
```
Comment (tkalejandro) [inline src/modules/Experience/components/Welcome/Welcome.tsx]: You can delete this if you want :-) Is just dummy.

**Review by tkalejandro — CHANGES_REQUESTED** [review]
Please check my comments :-)

File: src/modules/Experience/components/Welcome/Welcome.tsx (hunk 2 above)
Comment (Hicham2012) [reply src/modules/Experience/components/Welcome/Welcome.tsx:19]: Yeah, once we start editing the model on blender we will store it inside a dedicated folder.
(Just testing for now)

**Review by tkalejandro — APPROVED** [review]
Noice!

**Pull request merged**